src/ontology/mondo-edit.obo

```diff
@@ -513960,6 +513952,16 @@ relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0001-5208-3432
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6461" xsd:anyURI
 
+[Term]
```

Comment:
this addition and deletion (in line 513984) can be ignored, this is just Protege being weird and reordering the terms in the file

Reply:
This should not happen. If you to a load safe on master with Protege, do you get a diff?

Reply:
what is a safe load on master?

Reply:
do you mean open the master file in protege and do 'save as' and see if I get a diff? I will try that

Reply:
yep - I see that term reordered. @sabrinatoro was saying we need to run make norm after all the PRs I did to add new terms are merged